packages/blockchain-wallet-v4-frontend/src/modals/Brokerage/components/index.tsx

```diff
@@ -94,17 +94,34 @@ const BankRow = styled.div`
     justify-content: flex-start;
     align-items: center;
   }
+
+  ${props =>
+    props.onClick &&
+    css`
+      cursor: pointer;
+      * {
```

Comment:
trying to override the inner CSS? Check this out: https://styled-components.com/docs/faqs#how-can-i-override-styles-with-higher-specificity

Reply:
Yeah actually here we do not need to have specific cases if it's clickable or not since it's always clickable component